R/tidyr.R

```diff
@@ -53,3 +53,9 @@ tidyselect::starts_with
 #' @importFrom tidyselect last_col
 #' @export
 tidyselect::last_col
+#' @importFrom tidyselect any_of
```

Comment:
I think these can all actually be removed now, since tidyselect automatically provides an environment where they're available.

Reply:
Sure.

Btw, dplyr does reexport `any_of()` and `all_of()`. Do you also want to remove them there? Or, is it too late?

Reply:
I think you can remove all the tidyselect helpers now.

Reply:
Well, I guess people might have used `tidyr::starts_with()`, so we probably need to save that for a bigger release. That's more likely to be the case with dplyr, but I bet there's at least one CRAN package that has done it with tidyr too.

Reply:
Do I need to add a NEWS item for stopping re-exporting them?

Reply:
Sorry, I meant `starts_with()` as an example; we should probably keep all the existing exported tidyselect functions so we can get this PR into the existing patch release. And then remove them for the next minor release.

Reply:
Ah, got it! Sorry.